doc/api/n-api.md

```diff
@@ -3420,6 +3420,89 @@ This API returns the names of the enumerable properties of `object` as an array
 of strings. The properties of `object` whose key is a symbol will not be
 included.
 
+#### napi_key_collection_mode
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+```C
+typedef enum {
+  napi_key_include_all_prototypes,
+  napi_key_include_own_prototypes
```

Comment:
These should be
```C++
napi_key_include_all_properties,
napi_key_include_own_properties
```

IOW *properties*, not *prototypes*.

Reply:
As a reference, it's declared as `enum class KeyCollectionMode { kOwnOnly, kIncludePrototypes };` in v8.

Also, IMO there are no `own prototypes`, all prototypes in the chain are their own, right?